readme.md

```diff
@@ -1,7 +1,7 @@
 # HTML form
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_html-form/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_html-form/report/html_report/)
+- [DEMO LINK](https://flocky674.github.io/layout_html-form/)
+- [TEST REPORT LINK](https://flocky674.github.io/layout_html-form/report/html_report/)
```

Comment:
don't forget to add these links into pr's description

Reply:
how can i do that?